fedimint-testing/src/bin/fixtures.rs

```diff
@@ -152,10 +152,13 @@ async fn await_fedimint_block_sync() -> anyhow::Result<()> {
 
 async fn run_bitcoind() -> anyhow::Result<()> {
     let btc_dir = env::var("FM_BTC_DIR")?;
-
-    // spawn bitcoind
+    let current_dir = env::current_dir()?;
```

Comment:
We should not assume that `current_dir` will be the project root. You can use `get_project_root` instead to get the root directory of the project.

Reply:
wow, never knew of that function. interesting implementation.

Reply:
Thanks @m1sterc001guy , I'll implement that